docs/docs/sourcing-from-seams-cms.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Sourcing from Seams-CMS
+---
+
+## Overview
+
+[Seams-CMS](https://seams-cms.com) is a headless CMS system that allows you to easily manage content and data with either simple or complex structures and supports localization.
```

Comment:
```suggestion
[Seams-CMS](https://seams-cms.com) is a headless CMS system that allows you to manage content and data with either simple or complex structures and supports localization.
```

To help keep things welcoming and inclusive, try to [avoid words like "easy"](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#avoid-easy-and-simple).